RecipeBook/ViewModels/HomeScreenViewModel.swift

```diff
@@ -20,10 +20,15 @@ class HomeScreenViewModel {
         return userDefaultsName.getUserName()
     }
 
-    var numberOfItemsInSection: Int {
+    var categoriesNumberOfItemsInSection: Int {
         return categories.count <= 5 ? categories.count : 5
     }
 
+    var recommendationsNumberOfItemsInSection: Int {
+//        return recommendations.count <= 5 ? recommendations.count : 5
+        return 5
```

Comment:
Use the code you commented, you can never be sure that you have at least 5 meals coming from the server.